addons/point_of_sale/static/src/js/models.js

```diff
@@ -129,11 +129,15 @@ exports.PosModel = Backbone.Model.extend({
                         done.resolve();
                 },
                 function(statusText, url){
+                        done.resolve();
                         if (statusText == 'error' && window.location.protocol == 'https:') {
```

Comment:
`if (show_loading_error && statusText == 'error' && window.location.protocol == 'https:') {`

This will detect if the connection error happens before any PoS screens are loaded, so the instruction screen will only pop up at loading of the session.

Reply:
thanks, done now there is only one error notification at the start:
 https://youtu.be/vBitFgdh-kI